src/transaction/creation_common.rs

```diff
@@ -355,8 +358,57 @@ pub(super) mod paid_amount {
 
     const ARG_VALUE_NAME: &str = common::ARG_INTEGER;
 
-    const ARG_HELP: &str =
-        "The amount previously reserved to pay for the current transaction.";
+    const ARG_HELP: &str = "The amount previously reserved to pay for the current transaction.";
+
+    pub(in crate::transaction) fn arg() -> Arg {
+        Arg::new(ARG_NAME)
+            .long(ARG_NAME)
+            .required(false)
+            .value_name(ARG_VALUE_NAME)
+            .help(ARG_HELP)
+            .display_order(DisplayOrder::StrikePrice as usize)
+    }
+
+    pub fn get(matches: &ArgMatches) -> &str {
+        matches
+            .get_one::<String>(ARG_NAME)
+            .map(String::as_str)
+            .unwrap_or_default()
+    }
+}
+
+pub(super) mod standard_payment {
+    use super::*;
+    pub(in crate::transaction) const ARG_NAME: &str = "standard-payment";
+
+    const ARG_VALUE_NAME: &str = common::ARG_STRING;
+
+    const ARG_HELP: &str = "Standard payment.";
```

Comment:
nit: a more helpful help string would be nice here. Something like: `Flag to determine if this transaction uses standard or custom payment`.

Reply:
Updated in 041470958d8151e2cb405baffe724234ddc591b9